content/rs/administering/database-operations/importing-data.md

```diff
@@ -107,6 +107,11 @@ To specify to import from a local mount point on a node:
 
     For example: `/mnt/Public/backup.rbd`
 
+In order to use this option, the storage needs to be mounted in the same path on all cluster nodes.
+The import source folder needs to be owned by the redislabs user with write permissions.
```

Comment:
@yanivhershkovich Read permissions isn't enough to do the import?

Reply:
@bmansheim 
I based it on @yaeltzirulnikov confluence article - 
https://redislabs.atlassian.net/wiki/spaces/RED/pages/164364331/NAS+local+storage

Just rephrased the wording a bit.

Reply:
@yanivhershkovich the confluence article is about both import and export, that's why it mentions write permissions are needed. For import, read permissions are enough.